src/lib/plot/series/abstract-series.js

```diff
@@ -104,47 +100,50 @@ export default class AbstractSeries extends PureRenderComponent {
    * Mouse over handler for all series.
    * @param {boolean} useValue Use value handler if true.
    * @param {Object} d Value object
+   * @param {Object} event Event.
    * @private
    */
-  _mouseOverHandler(useValue, d) {
+  _mouseOverHandler(useValue, d, event) {
```

Comment:
@chrisirhc Event is passed from inside of each of the series. I made the change to make it more transparent.